scheds/include/scx/common.bpf.h

```diff
@@ -640,6 +640,23 @@ static inline u32 log2_u64(u64 v)
                 return log2_u32(v) + 1;
 }
 
+/*
+ * Return a value proportionally scaled to the task's priority.
+ */
+static inline u64 scale_task_fair(const struct task_struct *p, u64 value)
```

Comment:
Maybe rename it to `scale_by_task_weight()`? I'm not sure what `fair` means there.